waveform_editor/tendencies/base.py

```diff
@@ -79,9 +78,10 @@ class BaseTendency(param.Parameterized):
     )
     annotations = param.ClassSelector(class_=Annotations, default=Annotations())
 
-    def __init__(self, **kwargs):
+    def __init__(self, allow_zero_duration=False, **kwargs):
         self.line_number = kwargs.pop("line_number", 0)
         self.is_first_repeated = kwargs.pop("is_first_repeated", False)
+        self.allow_zero_duration = allow_zero_duration
```

Comment:
`allow_zero_duration` is a class property, so we can just make this a class variable instead of an instance variable?